src/sync/index.js

```diff
@@ -429,20 +456,67 @@ async function updateCentralizedOraclesPassedResultSetEndBlock(currentBlockChain
       {$set: {status:'OPENRESULTSET'}}, {multi: true});
     console.log('Updated COracles Passed ResultSetEndBlock');
   }catch(err){
-    console.error(`Error: updateCentralizedOraclesPassedResultSetEndBlock ${err.message}`);
+    console.error(`ERROR: updateCentralizedOraclesPassedResultSetEndBlock ${err.message}`);
   }
 }
 
+async function updateOracleInfo(db, oraclesNeedInfoUpdate) {
+  _.forEach(oraclesNeedInfoUpdate, async (oracleAddress, index) => {
+    let oracle;
+    try {
+      oracle = await db.Oracles.findOne({ 
+        address: oracleAddress 
+      });
+
+      if (!oracle) {
+        console.error(`ERROR: could not find Oracle ${oracleAddress} in db`);
+        return false;
+      }
+    } catch(err) {
+      console.error(`ERROR: find Oracle ${oracleAddress} in db, ${err.message}`);
+      return false;
+    }
+
+    let topic;
+    try {
+      topic = await db.Topics.findOne({
+        address: oracle.topicAddress
```

Comment:
we can pass topicAddress in array to save it one more query

Reply:
this is fixed in the latest changes. I get the topicAddress from the parsed Oracle obj.